components/src/core/Drawer/Drawer.tsx

```diff
@@ -76,6 +76,13 @@ export const Drawer: React.FC<DrawerProps> = (props) => {
         ...viewProps
     } = props;
 
+    const defaultProps: Partial<DrawerProps> = {
```

Comment:
We have removed in previous PR https://github.com/etn-ccis/blui-react-native-component-library/pull/590/files

Reply:
This is a little different. This isn't using the `Component.defaultProps` syntax — this is just a variable declaration inside the component. Declaring a variable with the default values allows us to use the default value correctly when props are used in multiple places without having to destructure `props` with those default values everywhere we use it. (i.e., when we set a default value during destructuring, we are not actually changing the values in `props` — we are just creating new variables based on the values. So if we pass the `props` variable to a function or somewhere else in the code, it does not have the default values in it)

E.g.,
```
<Component />

const defaultProps = {propName: true};
const {propName = true} = props;

console.log(propName); // true
console.log(props.propName); //undefined

console.log(({...defaultProps, ...props}).propName); // true
```